templates/zerver/help/enable-high-contrast-mode.md

```diff
@@ -1,25 +1,15 @@
-# Enable high contrast mode for accessibility
+# Enable high contrast mode
```

Comment:
While we're at it, let's rename the article to "High contrast mode". (no Enable)